build_scripts/RedHat+Fedora/data/retroshare.desktop

```diff
@@ -3,8 +3,8 @@ Encoding=UTF-8
 Version=1.0
 Name=RetroShare
 Comment=Securely communicate with your friends
-Exec=/usr/bin/retroshare %U
-Icon=/usr/share/pixmaps/retroshare.xpm
```

Comment:
Is this standard to not use an explicit path? It seems pretty risky to me: the executable that will be launched may not be the one you think.

Reply:
It is good practice to use the absolute path, to be on the safe side, it is possible to use just the name.